src/RosApiNameConstants.h

```diff
@@ -28,6 +28,7 @@
 #define SERVICE_NAME_STOP_TRAJ_MODE "stop_traj_mode"
 #define SERVICE_NAME_QUEUE_TRAJ_POINT "queue_traj_point"
 #define SERVICE_NAME_SELECT_MOTION_TOOL "select_motion_tool"
+#define SERVICE_NAME_LIST_JOBS "job/list"
```

Comment:
I've opted to create a `job` namespace here, as the full implementation of #278 will add 4 more services. The namespace groups them all using a single `job` prefix.

Reply:
Is `job` too generic? Would `inform_job` be better, or too verbose/unnecessary?

Reply:
We could go for `inform_job`, but are there other types of jobs? If not, I'd suggest `job` would be sufficient.

I've also considered `program` and other more generic names (`robot_program`, `native_program`, `oem_program`, etc) -- so as to not tie this to Yaskawa terminology immediately -- but couldn't come up with something that really stuck.

Reply:
Thinking about it a bit more, I believe I do actually prefer `inform_job` instead of just `job`. It's indeed much clearer from a ROS API perspective.

I'll update the PR.